README.md

```diff
@@ -215,7 +215,85 @@ newFetchInstance(`https://${server}/${path}/?payload=` + payload)
 
 ## Use Cases
 
-> *Adding use cases is a WIP - if you find this proposal useful (or just want to explore some potential use cases), help by visiting [#4](https://github.com/weizman/Realms-Initialization-Control/issues/4) and share your use case to push this proposal forward!*
+Here are some use cases introduced by the community which led to composing this proposal.
+
+> _Please note that on our attempt to list use cases, we link them to real life examples by providing names of companies and/or products - none of them has actively reached out and provided their feedback on this proposal, but are merely given as use cases examples (unless clearly stated so at the top of each use case)._
+
+### Safe composability (sandboxing / confinement)
+
+> _This use case is approved by [Shopify](https://github.com/shopify) and [MetaMask](https://github.com/metamask)_
```

Comment:
I would go with something significantly more succinct and generic:

"""
The ability to safely embed untrusted code in a safe way is important for composability. Platforms can use it to allow their users as well as third party providers to enhance their provided functionality, and provide value to end users.

While the web platform allows such safe embedding through cross-origin iframes or workers, there are cases where same-origin untrusted-code embedding is required. E.g., in cases where the untrusted code may not be custom tailored to a sandboxed environment. 
In such cases, in order to guarantee the end user's security, platforms will constrain the capabilities that are available to the untrusted code by overriding native prototypes.

At the same time, it's hard to ensure that the overriding scripts are first to run in the embedded contexts, and determined attackers can use that to escape the sandbox under certain conditions. See [the same origin concern](https://weizmangal.com/2023/09/28/the-same-origin-concern/) for more details on that.

This proposal aims to solve this, by enabling the top-level document to guarantee running an initial script in every new same-origin realm, and prevent races that allow for sandbox escapes.
"""

Reply:
cff6cb7